conda-recipes/llvmdev_manylinux/build.sh

```diff
@@ -62,7 +62,7 @@ cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
       -DLLVM_ENABLE_Z3_SOLVER=OFF \
       -DLLVM_OPTIMIZED_TABLEGEN=ON \
       -DCMAKE_POLICY_DEFAULT_CMP0111=NEW \
-      -DCOMPILER_RT_BUILD_BUILTINS=ON \
+      -DCOMPILER_RT_BUILD_BUILTINS=OFF \
```

Comment:
Would 

```suggestion
      -DCOMPILER_RT_BUILD_BUILTINS=ON \
      -DCOMPILER_RT_ENABLE_IOS=OFF \
```

work around the issue and enable compiler-rt to be built?

Reply:
The suggestion worked. Compiler-rt is reenabled in https://github.com/numba/llvmlite/pull/1104/commits/135be02cd38b0485dfb2ee55093a54aae803ef85